docs/2021-ga-release-blog-post.md

```diff
@@ -0,0 +1,76 @@
+4. [@azure/arm-sql](https://www.npmjs.com/package/@azure/arm-sql)
+5. [@azure/arm-resources](https://www.npmjs.com/package/@azure/arm-resources)
+6. [@azure/arm-policy](https://www.npmjs.com/package/@azure/arm-policy)
+7. [@azure/arm-managedapplications](https://www.npmjs.com/package/@azure/arm-managedapplications)
+8. [@azure/arm-links](https://www.npmjs.com/package/@azure/arm-links)
+9. [@azure/arm-locks](https://www.npmjs.com/package/@azure/arm-locks)
+10. [@azure/arm-features](https://www.npmjs.com/package/@azure/arm-features)
+
+These libraries follow the new [Azure Library guidelines](https://azure.github.io/azure-sdk/general_introduction.html) that were written to provide an improved and consistent development experience across Azure services. The [TypeScript & JavaScript specific guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html) that are also followed ensure that these libraries have a natural and idiomatic feel to the JavaScript and TypeScript ecosystem. These libraries have been generated using the new [Autorest Typescript Library Generator](https://www.npmjs.com/package/@autorest/typescript). They provide new authentication mechanisms, better paging and long running operations experience.. We encourage you to try out the libraries and provide your feedback.
+
+## Key concepts
+
+While a detailed list and usage of the new features can be found in the migration guide and the quick start guide can be used for first time users, below is an overview of the major changes in the new libraries
+
+### Authentication
+
+Previously, the Azure management libraries required credential objects created via the packages `@azure/ms-rest-nodeauth` or `@azure/ms-rest-browserauth` that used ADAL which reaches end of support in Jun 2022. The new Azure management libraries require credential objects from the new `@azure/identity` package that makes use of the [MSAL](https://docs.microsoft.com/azure/active-directory/develop/msal-overview) and have many more authentication options to support newer scenarios.
+
+- For the different authentication scenarios, see the [Azure Identity examples](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md)
+- Refer to the [migration guide to @azure/identity](https://github.com/Azure/ms-rest-nodeauth/blob/master/migrate-to-identity-v2.md) to update your existing application.
+
+### Promises vs callbacks
+
+To help with code readability, better handling of deferred and asynchronous computations and better error handling, our new guidelines encourage the use of promises over callbacks. Therefore, the overloads that took callbacks have been dropped in favor of promises.
+
+### PagedAsyncIterableIterator interface
+
+List operations now return a `PagedAsyncIterableIterator` interface that can be used to iterate over the results. This is a better user experience than before where a new method call had to made to access the next page. ```
```

Comment:
The guidance here https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md also covers quick sample of how to migrate to promises and the `PagedAsyncIterableIterator ` . Can we add a quick navigation to that section of the migration guidance so that readers can quickly understand how they can utilize these new feature?

Reply:
Done